providers/src/airflow/providers/fab/www/extensions/init_security.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Copy of `airflow/www/extensions/init_security.py`. I kept only `init_xframe_protection` method